pyaccel/accelerator.py

```diff
@@ -342,6 +342,7 @@ def __str__(self):
         rst += '\ncavity_on      : ' + str(self.trackcpp_acc.cavity_on)
         rst += '\nradiation_on   : ' + str(self.trackcpp_acc.radiation_on)
         rst += '\nvchamber_on    : ' + str(self.trackcpp_acc.vchamber_on)
+        rst += '\nlattice version: ' + str(self.trackcpp_acc.lattice_version)
```

Comment:
In case it already is a python string, you don't need the `str()` to convert it.

Reply:
You're right, thanks.